with-near-operation-file-preset/src/components/Feed/query.gql

```diff
@@ -0,0 +1,6 @@
+fragment FeedFields on Feed {
```

Comment:
Let's name this file `fragment.gql` because this is not query!

Reply:
aha, that's true.